lang/markdown/markdown.py

```diff
@@ -16,4 +16,6 @@
     "shell": "shell",
     "bash": "bash",
     "json": "json",
+    "are": "r",
```

Comment:
This looks like a separate change from the snippets. Based on my experience attending backlog sessions, the maintainers might ask you to make that a separate pull request.

Reply:
Thanks for the heads up. It is a fairly minor change and functions very similarly to a snippet "python block" rather than "snip code block"), so I'll leave it and hope.